pkg/vex/vex.go

```diff
@@ -15,7 +15,7 @@ type Config struct {
 }
 
 // FromPackageConfiguration generates a new VEX document for the Wolfi package described by the build.Configuration.
-func FromPackageConfiguration(buildCfg build.Configuration, vexCfg Config) (vex.VEX, error) {
+func FromPackageConfiguration(buildCfg *build.Configuration, vexCfg Config) (vex.VEX, error) {
```

Comment:
Why introduce a pointer here?

Reply:
Same, moving copies of these structs around will get flagged by the linter.

Reply:
Okay, ditto [this comment](https://github.com/wolfi-dev/wolfictl/pull/8#discussion_r1058645226) — e.g. now this function can panic during a nil pointer dereference